packages/app/src/App.tsx

```diff
@@ -69,10 +69,7 @@ const catalogRouteRef = createRouteRef({
 const routes = (
   <FlatRoutes>
     <Navigate key="/" to="/catalog" />
-    <Route
-      path="/catalog-import"
-      element={<ImportComponentRouter catalogRouteRef={catalogRouteRef} />}
-    />
+    <Route path="/catalog-import" element={<ImportComponentRouter />} />
```

Comment:
Not sure if any changes will be needed in plugin-api-docs as well, i.e. here?

https://github.com/backstage/backstage/blob/5df0714a84dc87561a434eaf23f5019e9623798a/plugins/api-docs/src/components/ApiExplorerPage/ApiExplorerPage.tsx#L36-L43